src/App.scss

```diff
@@ -0,0 +1,153 @@
+// variables
+$colorGreen: #cdc6ac;
+$lightGreen: #e0e1d8;
+$colorBrown: #a85d36;
+$colorPurple: #b0a19d;
+$quizFont: "IM Fell DW Pica", serif;
+
+body {
+  background-color: $colorGreen;
+  font-family: $quizFont;
+  margin: 0 auto;
+}
+
+h1 {
+  font-size: 62px;
+  font-style: italic;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+h2 {
+  font-size: 54px;
+  font-style: italic;
+  text-decoration: overline;
+  margin: 0;
+}
+
+.tv-header {
+  background-image: url(./assets/pattern_overlapping-hexagons_0.5_2_0-0_0_1__cdc6ac_ffffff.png);
+  //   background-repeat: no-repeat;
+  //   background-size: 100%;
+  .header-container {
+    display: flex;
+    justify-content: center;
+    .header-text {
+      width: 50%;
+      color: $colorBrown;
+      line-height: 5rem;
+      padding-top: 200px;
+      padding-left: 80px;
+    }
+  }
+  .text-container {
+    width: 50%;
+  }
+  .image-container {
+    width: 50%;
+    .header-image {
+      background-color: transparent;
+      max-width: 550px;
+      position: relative;
+      top: 6px;
+      left: 10px;
+    }
+  }
+}
+
+#page-container {
+  position: relative;
+  min-height: 100vh;
+}
+
+#content-wrap {
+  max-width: 1280px;
+  padding-bottom: 2.5rem; /* Footer height */
+  text-align: center;
+}
+
+.quiz-container {
+  width: 80%;
+  margin: auto;
+  background-color: $lightGreen;
+  margin-top: 80px;
+  margin-bottom: 80px;
+  padding: 70px 10px;
+  h3 {
+    font-style: italic;
+    font-size: 24px;
+    font-weight: 300;
+    padding: 10px 90px;
+  }
+}
+.button-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+  button {
+    width: 80%;
+    height: 50px;
+    margin: 10px;
+    border: none;
+    border-radius: 10px;
+    font-family: $quizFont;
+    font-size: 20px;
+    text-transform: uppercase;
+    &:hover,
+    &:focus {
+      cursor: pointer;
+      background-color: $colorGreen;
+      border: 2px solid $colorPurple;
+    }
+  }
+}
+
+.right-answer {
+  border: 5px solid green;
+}
+
+.wrong-answer {
+  border: 5px solid red;
+}
+
+.next-button {
+  width: 200px;
+  height: 45px;
+  margin: 20px 0;
+  float: right;
+  background-color: #f0f0f3;
+  border: none;
+  color: $colorBrown;
+  font-size: larger;
+  font-weight: 800;
+  border-radius: 10px;
+  box-shadow: inset -10px -10px 10px #ffffff70, inset 10px 10px 10px #aeaec020;
+  &:hover,
+  &:focus {
+    cursor: pointer;
+    background-color: $colorBrown;
+    color: #f0f0f3;
+    box-shadow: none;
+  }
+}
+
+.answer-container {
+  .youtube {
+    padding-top: 80px;
+  }
+}
+#footer {
```

Comment:
if it's just the footer, you don't need the ID tag #, I think you can just write "footer" without a class or ID